README.md

```diff
@@ -295,6 +295,39 @@ func main() {
 [Cloud Monitoring]: https://cloud.google.com/monitoring
 [Cloud Trace]: https://cloud.google.com/trace
 
+### Debug Logging
+
+The Go Connector supports optional debug logging to help diagnose problems with
+the background certificate refresh. To enable it, provide a logger that
+implements the `debug.Logger` interface when initializing the Dialer.
+
+For example:
+
+``` go
+import (
+    "context"
+    "net"
+
+    "cloud.google.com/go/cloudsqlconn"
```

Comment:
it should be `alloydbconn`

Reply:
@ttosta-google if you add a suggestion, you or PR author can then commit the suggestion directly from the UI, makes life easier